Doc/library/socket.rst

```diff
@@ -595,6 +595,26 @@ The following functions all create :ref:`socket objects <socket-objects>`.
    .. versionchanged:: 3.2
       *source_address* was added.
 
+.. function:: create_server_sock(address, *, dual_stack=True, queue_size=5, reuse_addr=True)
+
+   Convenience function which creates a TCP server bound to *address* (a
```

Comment:
If you want to support UDP please consider adding `allow_broadcast` argument as we have in asyncio: https://github.com/python/cpython/blob/master/Lib/asyncio/base_events.py#L1154-L1158

Reply:
I don't think this is worth adding because one can simply set `SO_BROADCAST` against the returned socket. `reuse_addr` and `reuse_port` parameters are different because they necessarily must be set before `bind()`. I would rather keep the API as simple as possible.

Reply:
Ok